### PR TITLE
fix(gtk): select cloned item. Fixes #585

### LINKED ISSUE #585: When you clone something, the original is still highlighted instead of the clone
## Classification:

UI/Usability

## Reproducibility:

Always

## Version

AutoKey version: 0.95.10

Used GUI (Gtk, Qt, or both): GTK

Installed via: Mint's Software Manager

Linux Distribution: Mint Cinnamon

## Summary

Summary of the problem.

## Steps to Reproduce (if applicable)

1. Right-click on a phrase or script in the left sidebar
2. Click "Clone Item"

## Expected Results

The clone should be instantly highlighted to start doing stuff to

## Actual Results

The original remains highlighted

## Notes

Sorry I suck at programming or else I'd try to help!

### PR DESCRIPTION
**Pull request by nevarek**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -142,7 +143,7 @@ Bug fixes
 - Both QT and GTK versions will reload hotkeys after a keymap change event.
 - Fix locking issue
 - Expose Alt_GR as a hotkey modifier on GTK.
-- (GTK) Fixed GUI lock-up, if multiple script error notifications are posted in quick succession. The notifications are now rate-limited and won’t post more than one notification per second. Fixes issue #383 
+- (GTK) Fixed GUI lock-up, if multiple script error notifications are posted in quick succession. The notifications are now rate-limited and won’t post more than one notification per second. Fixes issue #383
```
Comment (BlueDrink9) [inline CHANGELOG.rst:193]: Whitespace change. Ideally we wouldn't have trailing whitespace in the first place, but it is usually considered annoying to open PRs that change whitespace in areas that aren't related to your change.

(Only a small PR so doesn't make a big difference in this case, but keep it in mind. Editor tools sometimes do this automatically, so review your diffs)

**Comment by BlueDrink9** [comment]
Thank you so much for the PR!
This looks fine to merge, but first could you please reword the changelog entry. It is unclear from looking at it what the old behaviour was that you are fixing, and what it has been changed to. It is useful to reference the issue number (#585) in the changelog.
Also, please use a `(GTK)` tag at the start to make it clear this only applies to one GUI.

Also, if you edit the PR description to add "fixes #585", when it gets merged it will automatically close the relevant issue on github!

**Comment by BlueDrink9** [comment]
trs

File: CHANGELOG.rst (hunk 1 above)
Comment (nevarek) [reply CHANGELOG.rst:193]: Ah dang it, this must have been introduced in the rebase. I can remove the change, so I will.

**Comment by nevarek** [comment]
Thanks for explaining the commit workflow, I'll be sure to make these updates!

**Comment by josephj11** [comment]
While this stands on its own as a valid enhancement/fix, does the Qt front end also need attention?

@nevarek Welcome to our dev community! We really appreciate your contributions.

**Comment by BlueDrink9** [comment]
Looks like this isn't an issue in qt, just gtk

**Comment by josephj11** [comment]
@nevarek Any reason this didn't get finished? It appears to be a valuable fix that just needs cosmetic changes.

**Comment by nevarek** [comment]
> @nevarek Any reason this didn't get finished? It appears to be a valuable fix that just needs cosmetic changes.

I don't recall, sorry.

**Comment by josephj11** [comment]
@nevarek From your response, I assume you are no longer interested in completing this bug fix. I am going to ask another developer to take a look at this if you don't want to do it.

Thanks again for this PR. (Please don't delete it because we haven't copied it yet, etc.)

**Comment by BlueDrink9** [comment]
Known issue with tests. No real reason not to merge this.

**Pull request merged**